modules/ROOT/pages/index.adoc

```diff
@@ -98,11 +105,9 @@ See xref:dashboards.adoc[].
 //feature tier (for h2-h4 headings):
 include::partial$subsection-pills.adoc[tag="anypoint-pricing-core"]
 
-Track trends and add your own alerts to identify abnormal behavior and problems in your app network.
-
-image::intro-alerts.png["Alerts in Anypoint Monitoring"]
```

Comment:
Made this overview text more general so that it covers alerts on all instances of Anypoint Platform. The details are in the docs for the specific instances.